libs/npm/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "tno-core",
   "description": "TNO shared library",
-  "version": "0.0.123",
+  "version": "0.0.124",
```

Comment:
Speaking of linters, to avoid mine going crazy on me there is a quick trick if there are one off times you want to save without your auto linter running. `CTRL + K`, followed by `Ctrl + SHIFT + S`